src/main/java/com/textmessenger/service/PostServiceImpl.java

```diff
@@ -117,25 +111,17 @@ private Sort orderBy() {
   public void retwitPost(User user, Long postId) {
     Post retweet = new Post();
     Post original = postRepository.getOne(postId);
-    String login = original.getUser().getLogin();
+    User user1 = original.getUser();
     retweet.setUser(user);
     retweet.setParent(original);
     Post save = postRepository.save(retweet);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "save". ([squid:S1854](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=177&issues=AWXKdPh6lgbYnohY2kyh&open=AWXKdPh6lgbYnohY2kyh)

[ISSUE_KEY]: AWXKdPh6lgbYnohY2kyh